Pytest `test_debsums` fails (includes the GL pipeline)

**What happened**:
Currently, the new PyTest `test_debsums` fails and avoids the pipeline (e.g. [here](https://github.com/gardenlinux/gardenlinux/runs/6625845493?check_suite_focus=true#step:4:6346)) from being completed for several build unites:
```
=================================== FAILURES ===================================
6346
____________________________ test_debsums[exclude0] ____________________________
6347

6348
client = <helper.sshclient.RemoteClient object at 0xffffa57c97b0>
6349
exclude = ['/lib/systemd/system/dbus.socket', '/usr/share/pam-configs/cracklib']
6350

6351
    @pytest.mark.parametrize(
6352
        "exclude",
6353
        [
6354
            [
6355
                "/lib/systemd/system/dbus.socket",
6356
                "/usr/share/pam-configs/cracklib"
6357
            ]
6358
        ]
6359
    )
6360
    
6361
    def test_debsums(client, exclude):
6362
>       debsums(client, exclude)
6363

6364
client     = <helper.sshclient.RemoteClient object at 0xffffa57c97b0>
6365
exclude    = ['/lib/systemd/system/dbus.socket', '/usr/share/pam-configs/cracklib']
6366

6367
../features/base/test/test_debsums.py:15: 
6368
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
6369

6370
client = <helper.sshclient.RemoteClient object at 0xffffa57c97b0>
6371
debsums_exclude = ['/lib/systemd/system/dbus.socket', '/usr/share/pam-configs/cracklib']
6372

6373
    def debsums(client, debsums_exclude):
6374
        """Check the MD5 sums of installed Debian packages"""
6375
        utils.AptUpdate(client)
6376
        (exit_code, output, error) = client.execute_command(
6377
            "apt-get install -y --no-install-recommends debsums", quiet=True)
6378
        assert exit_code == 0, f"no {error=} expected"
6379
    
6380
        (exit_code, output, error) = client.execute_command("debsums -l",
6381
                                                            quiet=True)
6382
        assert exit_code == 0, f"no {error=} expected"
6383
    
6384
        assert output == '', \
6385
                        f"the following packages don't have md5sums: {output}"
6386
    
6387
        (exit_code, output, error) = client.execute_command("debsums -sc",
6388
                                                            quiet=True)
6389
        assert exit_code == 0 or exit_code == 2, f"no {error=} expected"
6390
    
6391
        changed = []
6392
        for line in error.splitlines():
6393
            if line.split()[3] not in debsums_exclude:
6394
                changed.append(line.split()[3] + " " + line.split()[4] +
6395
                    " " + line.split()[5] + " " + line.split()[6])
6396
    
6397
>       assert len(changed) == 0, \
6398
                    f"the following files have changes: {', '.join(changed)}"
6399
E       AssertionError: the following files have changes: /usr/share/runit/meta/ssh/installed (from openssh-server package)
6400

6401
changed    = ['/usr/share/runit/meta/ssh/installed (from openssh-server package)']
6402
client     = <helper.sshclient.RemoteClient object at 0xffffa57c97b0>
6403
debsums_exclude = ['/lib/systemd/system/dbus.socket', '/usr/share/pam-configs/cracklib']
6404
error      = 'debsums: missing file /usr/share/runit/meta/ssh/installed (from openssh-server package)\n'
6405
exit_code  = 2
6406
line       = 'debsums: missing file /usr/share/runit/meta/ssh/installed (from openssh-server package)'
6407
output     = ''
6408

6409
helper/tests/debsums.py:27: AssertionError
```

**What you expected to happen**:
Test and pipeline should be successful. 

**How to reproduce it (as minimally and precisely as possible)**:
Run the test pipeline including the `test_dubsums` on any `metal` platform.

--- a/features/base/test/test_debsums.py
+++ b/features/base/test/test_debsums.py
@@ -6,6 +6,7 @@ import pytest
     [
         [
             "/lib/systemd/system/dbus.socket",
+            "/usr/share/runit/meta/ssh/installed",
             "/usr/share/pam-configs/cracklib"
         ]
     ]

